inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/MessageQueueZoneSinkContext.java

```diff
@@ -52,6 +68,10 @@ public class MessageQueueZoneSinkContext extends SinkContext {
     private final CacheClusterConfigHolder cacheHolder;
     private final INLONG_COMPRESSED_TYPE compressType;
 
+    private final MQClusterConfig mqConfig;
+    private MonitorIndex monitorIndex;
+    private MonitorIndexExt monitorIndexExt;
```

Comment:
No code use the data of monitorIndex and monitorIndexExt.

Reply:
It's intended to print statistics to file, index.log.